static/app/components/pickProjectToContinue.tsx

```diff
@@ -30,8 +31,18 @@ function PickProjectToContinue({
   projects,
 }: Props) {
   let navigating = false;
+  let path = `${nextPath}?project=`;
 
-  const path = nextPath.includes('?') ? `${nextPath}&project=` : `${nextPath}?project=`;
+  if (nextPath.includes('?')) {
```

Comment:
Is this just parsing query string params from the location path? can we use location.query to get and set these params instead of using split and string interpolation?

Reply:
it can also be done with `location.query`, but I have to convert it to string because the component ContextPickerModal only accepts a nextPath of type string. Maybe this is something we can update in another PR

Reply:
I've updated the code.   😉